test/pipeline_test.dart

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:http/http.dart';
+import 'package:test/test.dart';
+// \TODO REMOVE
```

Comment:
I was doing the pull requests all in tandem so I didn't want to end up with a conflict with the exports. Will fix with a rebase.